.circleci/config.yml

```diff
@@ -0,0 +1,101 @@
+version: 2.1
+
+orbs:
+   win: circleci/windows@2.4.0
+
+workflows:
+  test:
+    jobs:
+      - build-linux
+      - test-linux:
+          name: Java 8 - Linux - OpenJDK
+          docker-image: circleci/openjdk:8
+          requires:
+            - build-linux
+      - test-linux:
+          name: Java 9 - Linux - OpenJDK
+          docker-image: circleci/openjdk:9
+          requires:
+            - build-linux
+      - test-linux:
+          name: Java 10 - Linux - OpenJDK
+          docker-image: circleci/openjdk:10
+          requires:
+            - build-linux
+      - test-linux:
+          name: Java 11 - Linux - OpenJDK
+          docker-image: circleci/openjdk:11
+          requires:
+            - build-linux
+      - build-test-windows:
+          name: Java 11 - Windows - OpenJDK
+
+jobs:
+  build-linux:
+    docker:
+      - image: circleci/openjdk:8
+    steps:
+      - checkout
+      - run: java -version
+      - run: ./gradlew dependencies
+      - run: ./gradlew jar
+      - run: ./gradlew checkstyleMain
+      - persist_to_workspace:
+          root: build
+          paths:
+            - classes
+
+  test-linux:
+    parameters:
+      docker-image:
+        type: string
+      with-coverage:
```

Comment:
Is the plan to use `with-coverage` and `check-javadoc` later? They don't seem to be used at the moment.

Reply:
Oh— no, those are copy-pastes that aren't relevant for this.